src/main/java/com/giftmaseya/conveyorservice/controller/ConveyorController.java

```diff
@@ -15,24 +18,23 @@
 import javax.validation.Valid;
 import java.util.List;
 
+@Api(value = "REST APIs for the conveyor-resources")
 @RestController
 @RequestMapping("/conveyor")
+@AllArgsConstructor
```

Comment:
Here and in another places: It is better to use @RequireArgsConstructor to inject dependencies

Reply:
Yes. I used Swagger.